static/public/index.html

```diff
@@ -4245,3599 +7239,2764 @@ <h3 class="text-base font-medium text-dark">25%</h3>
+                        </svg>                  
+                    </div>
+                    <div class="absolute top-7 mt-0.5 left-64">
+                      <svg class=" " width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                      <path d="M6.66681 10.3906L4.47148 8.19527L3.52881 9.13794L6.66681 12.2759L13.1381 5.8046L12.1955 4.86194L6.66681 10.3906Z" fill="#07B255"/>
+                      </svg>
+                    </div>
+                </div>  
+
+
+                <div class="relative">
+                    <label class="block font-light text-dark-grey text-xs ">Label</label>
+                    <input type="text" disabled placeholder="Enter text" class="bg-white placeholder-grey-shd4 w-72 h-10 p-3 pl-9 rounded-md border-2 border-solid border-grey-shd7">
+                    <div class="absolute top-7 mt-0.5 left-3 ">
+                    <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M0.999801 7.66669H1.66647V9.00002V12.3334C1.66647 13.0687 2.26447 13.6667 2.9998 13.6667H4.9998H8.9998H10.9998C11.7351 13.6667 12.3331 13.0687 12.3331 12.3334V9.00002V7.66669H12.9998C13.2691 7.66669 13.5131 7.50402 13.6158 7.25536C13.7191 7.00602 13.6618 6.71936 13.4711 6.52869L7.47113 0.528691C7.21047 0.268024 6.78913 0.268024 6.52847 0.528691L0.528467 6.52869C0.337801 6.71936 0.280467 7.00602 0.383801 7.25536C0.486467 7.50402 0.730467 7.66669 0.999801 7.66669ZM5.66647 12.3334V9.00002H8.33313V12.3334H5.66647ZM6.9998 1.94269L10.9998 5.94269V9.00002L11.0005 12.3334H9.66647V9.00002C9.66647 8.26469 9.06847 7.66669 8.33313 7.66669H5.66647C4.93113 7.66669 4.33313 8.26469 4.33313 9.00002V12.3334H2.9998V9.00002V7.00002V5.94269L6.9998 1.94269Z" fill="#94ACB5"/>
+                        </svg>                  
+                    </div>
+                </div>  
+            
+
+
                 </div>
-                <button
-                  class="text-sm hidden md:inline-block font-medium text-white bg-primary opacity-24 pointer-events-none py-xxsv px-m rounded-full"
-                >
-                  Post
-                </button>
-              </div>
+
             </div>
-      
```

Comment:
1-letters should have grey color instead of dark
2-shouldn't have a margin on top